xml/obs_build_config.xml

```diff
@@ -827,7 +820,10 @@ Support: pax debbuild</screen>
        <varlistentry>
         <term><parameter>splitdebug:SUFFIX</parameter></term>
         <listitem>
-        <para>A second rpm-md repository is generated where all debuginfo and debugsource packages get moved to. The specified SUFFIX is added to the repository name.</para>
+        <para>A second rpm-md repository is generated where all debuginfo and debugsource packages get moved to. The specified SUFFIX is added to the repository name. For example:</para>
+        <screen>Repotype: rpm-md splitdebug:-debuginfo</screen>
+        <!-- oops, this doesn't seem to follow the syntax: -->
+        <!-- Repotype: TYPE[:OPTIONS] -->
```

Comment:
nevermind. When I realize that splitdebug is a repotype, then it all makes sense.